app/views/posts/index.html.erb

```diff
@@ -1,9 +1,41 @@
-<div>
-<h1> Posts </h1>
-<h2><%= link_to "Ask a Question", new_course_post_path(@id) %></h2>
- <% @posts.each do |p| %>
-   <h2><%= link_to p.title, course_post_path(@id, p.id) %></h2>
-    <p><%= p.description %><p>
-    <p><%= p.view %><p>
- <% end %>
+<div class="container-fluid">
```

Comment:
Please include the header partial in the next commit. So that the header appears on this page too. You can do this with 

`<header>
      <%= render "layouts/header"%>
   </header>`